### DISCUSSION TITLE
Old Twitter link at bottom of Read.me

### DISCUSSION THREAD

**Discussion by cuvtixo** [uBlockOrigin/uAssets#23771] [General]
I'm not sure what the right place for this is, but on the READ.ME there's a link to an R.Hill post, on what is now called "X" at the bottom **Similar-Purposed Blockers** on the phrase "similarly-purposed blockers" 
_Seriously: Do NOT use similar-purposed blocker(s) along with uBlock Origin: this will cripple uBO's ability to defuse anti-blocker mechanisms and its ability to minimize likelihood of site breakage. ("similar-purposed" = any other blocker making use of EasyList).
7:21 AM · Aug 26, 2018_
This was probably very cutesy in 2018: "Seriously"  Worse than useless now, practically a spam link for X. Please get rid of it!

**Comment by gorhill** [comment]
Fixed with <https://github.com/uBlockOrigin/uAssets/commit/3ffdeb43b4d5a4d9955230118f17b96f12a37d44>
